server/middleware/middleware.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+
+app.use('/user/:id', (req, res, next) => {
+    console.log('Request URL: ' + req.originalUrl)
+    next();
+}, (req, res, next) => {
+    console.log('Request Type:', req.method)
+    next(); 
+});
+
```

Comment:
Good job with the initial work on this @vmaineng !


This one just needs to be one function so that we can use it for all endpoints. 

```
const logger = (req, res, next) => {
console.log(${req.method} ${req.protocol}://${req.get('host')}${req.originalUrl})
next()
}

module.exports = logger
```

Also, let's rename this file to be called `logger.js`

Reply:
I pushed new updates - renamed the file to logger.js and compressed the functions into one. Please review and let me know what your thoughts are. Thanks!